test/c/CMakeLists.txt

```diff
@@ -1,4 +1,27 @@
+if (NOT MSVC)
+	set (CMAKE_PREFIX_PATH "$ENV{BOOST_ROOT}")
```

Comment:
I'm not sure if we should force people to set BOOST_ROOT as they env variable. If we can remove this lines, we can specify BOOST_ROOT from command line:

```bash
cmake -DCMAKE_PREFIX_PATH=$BOOST_ROOT .
```